agent/api/eni/eni.go

```diff
@@ -181,6 +184,25 @@ func (eni *ENI) GetHostname() string {
 	return eni.PrivateDNSName
 }
 
+// GetLinkName returns the name of the ENI on the instance.
+func (eni *ENI) GetLinkName() string {
```

Comment:
If this method is going to be called concurrently, we might need to protect this with a mutex.

Reply:
+1, please add RW lock here